cloudesire-api-client-dto/src/main/java/com/cloudesire/platform/apiclient/dto/model/dto/UserCompanyDTO.java

```diff
@@ -1,10 +1,22 @@
 package com.cloudesire.platform.apiclient.dto.model.dto;
 
 import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+
+import java.util.Date;
 
 @ApiModel( description = "Company data of a business customer" )
 public class UserCompanyDTO extends CompanyDTO
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Override the "equals" method in this class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS2160)